dev/minikube/BUILD.bazel

```diff
@@ -1,6 +1,33 @@
 package(default_visibility = ["//visibility:public"])
 
-sh_binary(
+load("//rules/helm:def.bzl", helm_template = "template")
+load("//rules/kubectl:def.bzl", kubectl_apply = "apply", kubectl_delete = "delete")
+load(":def.bzl", "minikube_start")
+
+NAMESPACE = "scf"
+
+minikube_start(
     name = "start",
-    srcs = ["start.sh"],
+)
+
+helm_template(
+    name = "rendered_scf",
+    set_values = {
+        "system_domain": "192.168.99.100.xip.io",
```

Comment:
Same as before - move this to a values file.
Also, this calculation of this "default dev values yaml" can happen in a separate target.

Separation of concern is achieved nicely:
- targets to build the helm chart
- targets to create development environments
- targets to generate configuration for well-known environments
- targets to install

Reply:
I'll skip the targets to generate configurations for well-known environments as the requirements for this is not well-defined yet.